src-ui/src/app/components/document-detail/document-detail.component.html

```diff
@@ -5,7 +5,7 @@
       <div class="input-group-text" i18n>of {{previewNumPages}}</div>
     </div>
 
-    <button type="button" class="btn btn-sm btn-outline-danger me-2 ms-auto" (click)="delete()">
+    <button type="button" class="btn btn-sm btn-outline-danger me-2 ms-auto" (click)="delete()" [disabled]="!userIsOwner">
```

Comment:
I think it should not be bound to being the owner but to documents-delete permission

Reply:
I think this one should be left as-is, owners can grant 'edit' permissions but only owners can delete their own docs, I think thats safer behavior, at least for now